src/HeaderBar.vala

```diff
@@ -43,6 +45,32 @@ public class Mail.HeaderBar : Hdy.HeaderBar {
             _("Compose new message")
         );
 
+        var refresh_button = new Gtk.Button.from_icon_name ("view-refresh", Gtk.IconSize.LARGE_TOOLBAR) {
+            action_name = MainWindow.ACTION_PREFIX + MainWindow.ACTION_REFRESH,
+            halign = Gtk.Align.START,
+            can_focus = false
```

Comment:
That's probably more this:
```suggestion
            can_default = false
```

Reply:
Unfortunately this does not have the desired effect on my end. The button is still "activated" upon app start, even if `can_default` is set to false. No clue why 🤷

Reply:
Yeah, same as the filter we don't want to set `can_focus` to false because then you can't access this button with the keyboard which is bad